src/vs/workbench/contrib/tasks/browser/task.contribution.ts

```diff
@@ -496,6 +496,13 @@ configurationRegistry.registerConfiguration({
 			description: nls.localize('task.quickOpen.showAll', "Causes the Tasks: Run Task command to use the slower \"show all\" behavior instead of the faster two level picker where tasks are grouped by provider."),
 			default: false
 		},
+		'task.allowAutomaticTasks': {
+			type: 'string',
+			enum: ['on', 'auto', 'off'],
```

Comment:
You can use `enumDescription` or `enumMarkdownDescription` if you need more formatting here.

Reference:

https://github.com/microsoft/vscode/blob/7c78640d86e5de1ca0270912584d9d38beafeec1/src/vs/workbench/contrib/terminal/common/terminalConfiguration.ts#L55-L59